aw_qt/manager.py

```diff
@@ -28,8 +28,12 @@ def _discover_modules_in_directory(path: str) -> List["Module"]:
     modules = []
     matches = glob(os.path.join(path, "aw-*"))
     for match in matches:
-        if os.path.isfile(match) and os.access(match, os.X_OK):
-            name = os.path.basename(match)
+        filename = os.path.basename(match)
+        if not _validate_module(filename):
+            continue
+        # NOTE: os.access(match, os.X_OK) seems to always return True on Windows
+        elif os.path.isfile(match) and os.access(match, os.X_OK):
```

Comment:
I don't think this is the right way to do it...

On windows an executable should simply be an file which ends with ".exe" while on Linux all files with executable permissions should be considered an executable.

.desktop files should not have executable permissions, .service files should not have executable permissions and neither should .manifest files on a system with proper permissions set.

Reply:
Good point. Should probably be changed to do as you say: only accept `.exe` files on Windows and on other systems only those with the executable bit set (although `.so` files do too, so but hopefully they'll be excluded by the `aw-*` filter).

However, `.desktop` files should have the executable bit set in some circumstances (although it doesn't in the zip): https://askubuntu.com/questions/419610/permission-of-a-desktop-file